src/lib/components/checkbox-input.svelte

```diff
@@ -1,8 +1,9 @@
 <script lang="ts">
 	export let checked: boolean;
+	export let onChange: (event : any) => void = () => {};
 </script>
 
 <label class="input px-2 cursor-pointer flex flex-row items-center gap-1">
-	<input type="checkbox" class="accent-accent bg-accent text-white" bind:checked />
+	<input type="checkbox" class="accent-accent bg-accent text-white" bind:checked on:change={onChange} />
```

Comment:
if you just do `<input on:change />` the event will be forwarded so you can do `<CheckboxInput on:change={etc} />`. The TagToggle component has an example of this.

Reply:
Forwarded it instead